packages/gen-ai-hub/src/orchestration/orchestration-client.ts

```diff
@@ -0,0 +1,50 @@
+import { HttpDestination } from '@sap-cloud-sdk/connectivity';
+import { CustomRequestConfig } from '@sap-cloud-sdk/http-client';
+import { DefaultApi } from './api/default-api.js';
+import {
+  CompletionPostRequest,
+  CompletionPostResponse
+} from './api/schema/index.js';
+
+/**
+ * Input Parameters for GenAI hub chat completion.
+ */
+export type GenAiHubCompletionParameters = Pick<
+  CompletionPostRequest,
+  'orchestration_config' | 'return_module_results'
+>;
+
+/**
+ * Get the orchestration client.
+ */
+export class GenAiHubClient {
+  destination: HttpDestination;
+  constructor(destination: HttpDestination) {
```

Comment:
Follow up: when orchestration GA, how to get information like URL, authentication related.
Once we know that, we can reconsider the design.

Reply:
Follow up: investigate, when a JWT is needed for what reasons.